tests/core/Did.spec.ts

```diff
@@ -0,0 +1,8 @@
+import Did from '../../lib/core/Did';
+
+describe('DID', async () => {
+  it('isDid() should return false if DID method is not in the DID.', async () => {
+    const isDid = Did.isDid('did:abcdefg');
+    expect(isDid).toBeFalsy();
+  });
+});
```

Comment:
coud add a positive test showing that 'did:abc:def' is a Did

Reply:
DID.ts has 100% code coverage. The positive case has been hit 128 times by the tests.